package.json

```diff
@@ -13,7 +13,50 @@
     "bin": {
         "azle": "./src/build/index.ts"
     },
-    "main": "./src/lib/stable/index.ts",
+    "exports": {
```

Comment:
There is a companion property to `exports` called `imports`. See this documentation: https://nodejs.org/api/packages.html#package-entry-points

They show examples of how to do `internal` functionality, similar to what we're trying to accomplish with `_internal`.

Can you please study this? It might be a more elegant and best-practices way to accomplish what we want.

Reply:
It's slightly different, imports is for things that are only internal. Our internal things are used in outside project. But this is a good way to clean up some of the crazier relative paths. Implementing it on a large scale would be more of a 1.0 issue if we want to pursue it